lib/terminal.js

```diff
@@ -22,18 +29,20 @@ export default {
 
     this.disposables.add(atom.commands.add('atom-workspace', {
       'terminal:open': this.open.bind(this),
+    }));
+    this.disposables.add(atom.commands.add('terminal-view', {
```

Comment:
Separated these commands from workspace because all the callbacks expect valid terminal panes to be selected